assets/js/esDevelopersItaliaQuery.js

```diff
@@ -655,15 +655,15 @@ esDevelopersItaliaQuery.prototype.getSortQuery = function () {
 
   switch (value) {
     case 'az':
-      sort.push({
-        'name.raw': { 'order': 'asc' }
-      });
+      sort.push(
+        { 'publiccode.description.it.localizedName.keyword': { 'order': 'asc', unmapped_type: 'keyword' } },
```

Comment:
do we still need `unmapped_type`?

Reply:
We do because the query / filtering can be performed against indexes that don't have that particular mapping (eg. administrations, platforms). That happens when the search page returned multiple types and you want to filter on those. 

With no `unmapped_type`, the missing mapping error makes all the results from that index disappear. We need more work in the future to have those actually sorted though.

Reply:
got it :+1: